manifests/apache.pp

```diff
@@ -68,20 +67,23 @@
   $apache_cert = "${pki_dir}/certs/katello-apache.crt"
   $apache_key  = "${pki_dir}/private/katello-apache.key"
   # This variable is unused but considered public API
-  $apache_ca_cert = $certs::katello_server_ca_cert
```

Comment:
We use this, which is why there was an explicit comment above it to keep it in.
https://github.com/theforeman/puppet-katello_devel/blob/596745918fa758f209dbffb81d673c6c011186c9/manifests/apache.pp#L9
https://github.com/theforeman/puppet-foreman_proxy_content/blob/1c787b10b0d2f3bb4dbe1a7b7095feab03786a07/manifests/reverse_proxy.pp#L57

Reply:
I fixed it over here -- https://github.com/theforeman/puppet-certs/pull/420 which has that particular commit broken out.